src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -124,6 +124,7 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
                     },
                     (assembly, typeName, ignoreCase) =>
                     {
+                        _logger.ScriptStartUpLoadedStartUpExtension(startupExtensionName, assembly.GetName().Version.ToString());
                         return assembly?.GetType(typeName, false, ignoreCase);
                     }, false, true);
```

Comment:
While you're in here -- can you remove the constructor above? (I can't actually comment on it since it's not really part of the PR :-)). It got us into trouble here where we didn't actually do any logging in prod:

```
   public ScriptStartupTypeLocator(string rootScriptPath, IExtensionBundleManager extensionBundleManager)
            : this(rootScriptPath, NullLogger.Instance, extensionBundleManager)
        {
        }
```
It may break some tests, but I'd rather each test pass `NullInstance.Logger` so it forces us to always pass a logger in here.

Reply:
And additionally -- make the other constructor take an `ILogger<ScriptStartupTypeLocator>` rather than an `ILogger`. Just in case we ever switch it to use DI we won't have to change this again.